ambry-replication/src/test/java/com.github.ambry.replication/ReplicationTest.java

```diff
@@ -1250,4 +1256,19 @@ public void checkInConnection(ConnectedChannel connectedChannel) {
     public void destroyConnection(ConnectedChannel connectedChannel) {
     }
   }
+
+  /**
+   * No-op StoreKeyConverter. StoreKeys get paired with themselves
+   */
+  class NoOpConverter implements StoreKeyConverter {
```

Comment:
there is the `StoreKeyConverterImplNoOp` that does this

Reply:
I couldn't use it because it is defined in the server package and including it creates a circular dependency. @dharju is already making a change to move it to the `Store` and will reuse that.

Don't want to use the Mock one that was introduced, as the requirement here is for a simple, stateless converter.

Reply:
The mock should be exactly what you need now with some recent changes by @dharju

Reply:
Done.